deploy/backup.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: ps.percona.com/v1alpha1
+kind: PerconaServerMySQLBackup
+metadata:
+  name: backup5
+spec:
+  clusterName: cluster1
+  storageName: s3-west
```

Comment:
please make this `s3-us-west` because that is how it is placed in cr.yaml

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/77/commits/b772f58a7b9d2e1df3ec49632b9b66ba6e80d9f9